util/cmake/ctd/plugin.properties.in

```diff
@@ -1,14 +1,14 @@
 # the package of the plugin
-pluginPackage=de.seqan
+pluginPackage=@CTD_PLUGIN_PACKAGE@
 
 # the name of the plugin
-pluginName=SeqAn
+pluginName=@CTD_PLUGIN_NAME@
 
 # the version of the plugin
 pluginVersion=@CF_SEQAN_VERSION@
 
 # the path (starting from KNIMEs Community Nodes node)
-nodeRepositoyRoot=community
+nodeRepositoyRoot=community/@CTD_REPO_SUB_FOLDER@
```

Comment:
Don't we need a default value for CTD_REPO_SUB_FOLDER?
It seems CTD_PLUGIN_PACKAGE and CTD_PLUGIN_NAME have default values in cmake. 
But this particular flag is not.